Game/Source/ThirdPersonShooter/Game/TPSGameState.cpp

```diff
@@ -5,6 +5,7 @@
 #include "TPSLogging.h"
 #include "Teams/TPSTeams.h"
 #include "UnrealNetwork.h"
+#include "Engine/Engine.h"
```

Comment:
What was it that you had to include from the engine header? is there a more isolated header available for that feature?

Reply:
I need the Exec function, would/should we cast to a FExec and run via that?

Reply:
I think FExec is defined in `Runtime/Core/Public/Misc/Exec.h` if you can use that header then I think it will reduce the include bloat

Reply:
It seems I don't need it, visual studio complains but it builds anyway /shrug